azure-pipelines.yml

```diff
@@ -455,10 +459,14 @@ stages:
                 MSBUILD_PATH: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\MSBuild\\Current\\Bin\\MSBuild.exe"
                 PLATFORMTOOLSET: "v140"
                 WINDOWSSDKTARGET: "10.0.17763.0"
+                JDKVERSION: "1.17"
           variables:
             MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
             MAVEN_OPTS: '-Xmx3072m -Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
-          pool: '.Net Bubble - GCP'
+          pool: 
+            name: '.Net Bubble staging'
+            demands:
+              - PR_NUMBER -equals 154
```

Comment:
This needs to be removed

Reply:
I will revert after the image is available in the `.Net Bubble - GCP` pool.